addons/hw_drivers/iot_handlers/drivers/PrinterDriver_W.py

```diff
@@ -150,6 +150,11 @@ def open_cashbox(self, data):
             self.print_raw(drawer)
 
     def _action_default(self, data):
-        self.print_report(b64decode(data['document']))
+        document = b64decode(data['document'])
+        data_format = data.get('data_format', None)
```

Comment:
maybe do:
data_format = data.get('data_format', 'pdf')
 to keep the same behavior as before if the  'data_format' is not provided.